stablehlo/conversions/tosa/tests/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 OpenXLA Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+configure_lit_site_cfg(
+        ${CMAKE_CURRENT_SOURCE_DIR}/lit.site.cfg.py.in
+        ${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg.py
+        MAIN_CONFIG
+        ${CMAKE_CURRENT_SOURCE_DIR}/lit.cfg.py
+)
+
+set(STABLEHLO_TOSA_TEST_DEPENDS
+        FileCheck count not
+        stablehlo-tosa-opt
+)
+
+add_lit_testsuite(check-stablehlo-tosa-lit "Running the stablehlo-tosa regression tests"
+        ${CMAKE_CURRENT_BINARY_DIR}
+        DEPENDS ${STABLEHLO_TOSA_TEST_DEPENDS}
+        )
+set_target_properties(check-stablehlo-tosa-lit PROPERTIES FOLDER "Tests")
+
+add_lit_testsuites(STABLEHLO_TOSA_OPT ${CMAKE_CURRENT_SOURCE_DIR} DEPENDS ${STABLEHLO_TOSA_TEST_DEPENDS})
+
+add_dependencies(check-stablehlo-tosa check-stablehlo-tosa-lit)
```

Comment:
Let's trim down this file a little bit? A few months ago, we did a bit of a cleanup of build files for FileCheck tests in this repo and discovered that some things were unused (e.g. the dependencies on `count` and `not`). 

Unless I'm missing something, I think something as minimal as https://github.com/openxla/stablehlo/blob/main/stablehlo/testdata/CMakeLists.txt might work?